### PR TITLE
add a new play - [random meme generator]

### PR DESCRIPTION
**Pull request by murtuzaalisurti**
# Description

A project to demonstrate the use of API to fetch random memes!

## Type of change

New Play Added

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by atapas** [comment]
One request,

For your play, can you please drop a readme like this one? It may help beginners to connect to the code faster:

https://github.com/atapas/react-play/blob/main/src/plays/social-card/Readme.md


Also, when it is ready to review, please tag me.

**Comment by murtuzaalisurti** [comment]
@atapas readme added

**Review by atapas — COMMENTED** [review]
You can add a cover by creating any relevant cover.png and drop it in your play folder. That will be great. Let me know if you need help.

**Comment by murtuzaalisurti** [comment]
> You can add a cover by creating any relevant cover.png and drop it in your play folder. That will be great. Let me know if you need help.

done ✨

File: public/index.html [hunk 1]
```diff
@@ -43,6 +43,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (atapas) [inline public/index.html:46]: Can we avoid font awesome or any other CSS library and use as much CSS by hand as possible? It is to make sure, we are not opinionated about fontawesome vs bootstrap vs material etc. Maybe in future, we will take a call as a community and get one.

Also for the icons, react-icons library is already in to make use of it.

File: src/plays/random-meme-generator/RandomMemeGenerator.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      console.error(err);
+    })
+  }
+
+  useEffect(() => {
+    getMeme();
+  }, [])
+
+  function handleClick(e) {
+    e.target.classList.contains('generate-btn') && getMeme();
```
Comment (atapas) [inline src/plays/random-meme-generator/RandomMemeGenerator.jsx:46]: Why is this check necessary?

**Review by atapas — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (murtuzaalisurti) [reply public/index.html]: Done!

File: src/plays/random-meme-generator/RandomMemeGenerator.jsx (hunk 2 above)
Comment (murtuzaalisurti) [reply src/plays/random-meme-generator/RandomMemeGenerator.jsx]: Removed it now, actually I was using 2 buttons back then, but now the function isn't needed anymore.

**Review by atapas — APPROVED** [review]
@murtuzaalisurti Changes are looking good.

**Pull request merged**